SofaKernel/framework/sofa/simulation/CMakeLists.txt

```diff
@@ -131,7 +131,10 @@ set(SOURCE_FILES
     InitTasks.cpp
 )
 
-
+# is this a compiler/linker version specific problem?
+if(CMAKE_SYSTEM_NAME STREQUAL Linux)
```

Comment:
Why ?

Reply:
This is a compiling fix for Linux.
This comes from a merge of PR #969  in this branch.